src/controllers/authController.js

```diff
@@ -0,0 +1,49 @@
+import jwt from 'jsonwebtoken';
+import models from '../models';
+import bcrypt from 'bcryptjs';
+const signToken = (id) =>
```

Comment:
Can you remove some methods from here and add them in their own files in /utils or /helpers?

Methods like createSendToken, SignToken, and comparePassword

You will use them imported from there. This will make your codes clear and more easily scannable.